ui/webui/README.rst

```diff
@@ -8,9 +8,44 @@ Make sure you have `npm` available (usually from your distribution package).
 These commands check out the source and build it into the `dist/` directory::
 
     git clone https://github.com/rhinstaller/anaconda.git
+    cd anaconda
+    ./autogen.sh && ./configure
     cd ui/webui
     make
 
+Note that this builds only the WebUI code. In most cases, the rest of Anaconda is needed too.
+
+Running the UI interactively
+----------------------------
+
+First, build RPM packages in the CI container, from root of the repository::
+
+    make -f ./Makefile.am anaconda-ci-build
+    make -f ./Makefile.am container-shell
+    ./autogen.sh && ./configure
+    make rpms
+
+Alternatively, you can install all the dependencies and build without using the container::
+
+    sudo ./scripts/testing/install_dependencies.sh -y
+    ./autogen.sh && ./configure
+    make rpms
+
+Note that if you went through this process already, you also need to remove the whole `ui/webui/bots/` directory before calling `make rpms`. Another pitfall is that the `bots` directory is a git repository and will not be removed with `git clean`.
```

Comment:
That's actually not true.

You don't need to remove `bots` in every run, in fact that's extremely inefficient.

Removing the `bots` was needed in your scenario, because I had just introduced the new image for `anaconda-webui` tests (`fedora-rawhide-boot` image). In the older `bots` checkout the image was not present, so it would just give `Image does not exist` or similar error. Now that the image exists, having an older checkout of the `bots`, just means that you could possibly run your tests against an older `rawhide boot.iso`, which is not problematic. The CI always will use the newest image, as the bots there is always created.

Of course there is the chance that the newer anaconda code does not work any more with older ISO, in that case, it will become pretty intuitive to realize that the `bots` need rebase, in order to use the new image.